### PR TITLE
[string] Add string case style

### PR DESCRIPTION
**Pull request by zekth**

### CODE REVIEW

File: strings/case_style.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+
+export enum caseStyle {
+  pascalCase = "pascalCase",
```
Comment (axetroy) [inline strings/case_style.ts]: Should we make an example for every `caseStyle`?

Developers know more intuitively which style they need

```suggestion
  pascalCase = "pascalCase", // eg. `LastName`
```

**Comment by ry** [comment]
Looks good from a code level. But why do you need this?

**Comment by zekth** [comment]
> Looks good from a code level. But why do you need this?

examples:
- Uploading files and having the same way of encoding filenames
- input validation
- For linter development

As you can see on this package https://www.npmjs.com/package/camelcase it's quite downloaded (really surprised on the amount of download)

**Comment by ry** [comment]
I've never used such a routine. It seems specialized to a linter. I would reject this addition unless you can find an example of it in Go's std lib...

**Comment by zekth** [comment]
Closing moved => https://github.com/zekth/deno-case-style/

**Pull request closed without merging**